internal/k8s/configuration_test.go

```diff
@@ -2332,6 +2334,10 @@ func TestAddGlobalConfiguration(t *testing.T) {
 	if diff := cmp.Diff(expectedProblems, problems); diff != "" {
 		t.Errorf("AddOrUpdateTransportServer() returned unexpected result (-want +got):\n%s", diff)
 	}
+	storedGC := configuration.GetGlobalConfiguration()
```

Comment:
A very close variation on this code is repeated 5 times. Worth pulling it out to avoid duplicate code?

Reply:
while that test code is verbose, it is easy to read and follow. I don't think it's worth spending time reducing the duplication.